source/scpd/Cpp.d

```diff
@@ -151,14 +153,14 @@ private pure nothrow @nogc @safe extern(C++) size_t cpp_unordered_map_length (K,
 
 /// Rudimentary bindings for std::unordered_map
 extern(C++, (StdNamespace))
-public struct unordered_map (Key, T, Hash = hash!Key, KeyEqual = equal_to!Key, Allocator = allocator!(pair!(const Key, T)))
+public struct unordered_map (Key, T, Hash = RandHasher!(Key), KeyEqual = equal_to!Key, Allocator = allocator!(pair!(const Key, T)))
```

Comment:
Did we get this wrong or it's a new thing in C++ ?

Reply:
Ping

Reply:
It's a new thing in SCP:

https://github.com/stellar/stellar-core/blob/master/src/util/UnorderedSet.h
https://github.com/stellar/stellar-core/blob/master/src/util/UnorderedMap.h

I haven't looked into why they switched things.